core/templates/pages/story-editor-page/modal-templates/story-editor-unpublish-modal.component.html

```diff
@@ -1,70 +1,78 @@
-             aria-haspopup="menu"
-             aria-expanded="false"
-             class="oppia-selected-reason-text">
-          {{ selectedReasonText }}
-        </div>
-        <div ngbDropdownMenu
-             class="oppia-unpublishing-dropdown-menu">
-          <div ngbDropdownItem
-               (click)="setReason('BAD_CONTENT')"
-               (keydown.enter)="setReason('BAD_CONTENT')"
-               class="oppia-unpublishing-dropdown-item"
-               tabindex="0">
-            {{ badContentReasonText }}
+    <div class="modal-body-actions">
+      <div class="oppia-reason-select">
+        <span>Select Reason:</span>
+        <div ngbDropdown class="oppia-unpublishing-dropdown">
+          <div ngbDropdownToggle
+               aria-label="Dropdown toggle to select unpublishing reason"
+               aria-haspopup="menu"
+               aria-expanded="false"
+               class="oppia-selected-reason-text">
+            {{ selectedReasonText }}
           </div>
-          <hr>
-          <div ngbDropdownItem
-               (click)="setReason('CHAPTER_NEEDS_SPLITTING')"
-               (keydown.enter)="setReason('CHAPTER_NEEDS_SPLITTING')"
-               class="oppia-unpublishing-dropdown-item"
-               tabindex="0">
-            {{ splitChapterReasonText }}
+          <div ngbDropdownMenu
+               class="oppia-unpublishing-dropdown-menu">
+            <div ngbDropdownItem
+                 (click)="setReason('BAD_CONTENT')"
+                 (keydown.enter)="setReason('BAD_CONTENT')"
+                 class="oppia-unpublishing-dropdown-item"
+                 tabindex="0">
+              {{ badContentReasonText }}
+            </div>
+            <hr>
+            <div ngbDropdownItem
+                 (click)="setReason('CHAPTER_NEEDS_SPLITTING')"
+                 (keydown.enter)="setReason('CHAPTER_NEEDS_SPLITTING')"
+                 class="oppia-unpublishing-dropdown-item"
+                 tabindex="0">
+              {{ splitChapterReasonText }}
+            </div>
           </div>
         </div>
       </div>
-    </div>
 
-    <div class="oppia-warning-and-confirmation">
-      <div class="oppia-warning-container">
-        <span>Warning:</span>
-        <div class="bad-content-warning oppia-warning-text"
-             *ngIf="unpublishingReason=='BAD_CONTENT'"
-             tabindex="0"
-             role="alert">
-          “Please edit the existing explorations instead of adding/deleting new ones, to avoid inadvertently impacting learners' progress”
+      <div class="oppia-warning-and-confirmation">
+        <div class="oppia-warning-container">
+          <span>Warning:</span>
+          <div class="bad-content-warning oppia-warning-text"
+               *ngIf="unpublishingReason=='BAD_CONTENT'"
+               tabindex="0"
+               role="alert">
+            “Please edit the existing explorations instead of adding/deleting new ones, to avoid inadvertently impacting learners' progress”
```

Comment:
You don't need the quotes here. End with a period.

Reply:
Done